README.md

```diff
@@ -1,7 +1,26 @@
 # Task List
 
+
 Let's build a Task List in Rails! We will solve the problem of tracking tasks in a web application. This project will enable us to keep track of and persist, add, edit and remove tasks. 
 
+## Frequently Asked Questions
+
+### Due October Seventh!?
+
+**THIS IS COHORT 12's TASK LIST.  Are you in Cohort 13?  If so you probably meant to use [Cohort 13's Task List](https://github.com/Ada-C13/task-list/).**
+
+### Wait, I'm in Cohort 13 how did I get here then?
+
+That's our bad, we forgot to update the calendar link from last time to point at this Cohort's fork of task-list.  Sorry about that!
+
+### Oh no, I forked this already!
+
+Don't panic!  You will need to [delete your old C12 fork](https://help.github.com/en/github/administering-a-repository/deleting-a-repository) and create a new C13 fork.  If you haven't made any commits already then you're good to go after doing this.
+
+#### Umm... what if I already made some commits?
```

Comment:
I've tried to walk a student through this process before and ended up realizing 15 minutes in that I don't even know how to do this. Could we either (1) change this to just say "it's ok, for this project instructors will look for PRs in the C12 fork as well" or (2) writeup a process for _how_ to do this?

Reply:
Honestly I'm hoping that they just haven't done much.

I think the easiest solution will be to have them make some patch files and then apply them to a fresh copy of the repo.  Like here: https://makandracards.com/makandra/2521-git-how-to-create-and-apply-patches

The other thing that makes it easy is if they have only done a single controller so far or something then they can just copy that one file over.

Is that enough?  I'm really just hoping that people that forked the wrong repo haven't started yet.

Reply:
I don't hate the backup solution of just looking at both though.

Reply:
Actually, I think I'm just going to update it to say we'll grade it but for them to let us know.